lib/trino-hive-formats/pom.xml

```diff
@@ -113,6 +123,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-main</artifactId>
```

Comment:
should I make own version?

Reply:
I think we should move it to the SPI. When you do that, switch the TestNG `assertEquals` usage to AssertJ, since we're trying to move away from TestNG in new code.

Reply:
Getting the rolling IntelliJ errors on the refactor. I'll ping you offline on what you think we should do here. 
```
Field ColorType.COLOR, referenced in method BlockAssertions.createColorRepeatBlock(int, int), will not be accessible in module trino-spi  Field TestingConnectorSession.SESSION, referenced in method BlockAssertions.getOnlyValue(Type, Block), will not be accessible in module trino-spi  Field IpAddressType.IPADDRESS, referenced in method BlockAssertions.createRandomBlockForType(Type, int, float), will not be accessible in module trino-spi
```